Mathlib/RingTheory/Localization/FractionRing.lean

```diff
@@ -542,7 +542,7 @@ theorem nontrivial_iff_nontrivial : Nontrivial R ↔ Nontrivial S := by
   rcases h' with ⟨_, _⟩ | ⟨_, _⟩
   · obtain ⟨c, hc⟩ := h.exists_of_eq (x := 1) (y := 0) (Subsingleton.elim _ _)
     simp at hc
-  · apply (h.map_units 1).ne_zero
+  · apply (h.map_units _ 1).ne_zero
```

Comment:
I don't think we should change this, so `S` should be implicit in the original class. (in case you don't know, there's a new feature where you can change the explicitness of parameters inside each field.)